finch/default.cfg

```diff
@@ -18,7 +18,8 @@ language=en-US,fr
 [finch]
 subset_threads = 1
 datasets_config = datasets.yml
-default_dataset = candcs-u5
+default_dataset = candcs-u6
+xclim_modules = humidex;streamflow
```

Comment:
Hum, it seems the ";" can be used as a comment marker in the config (https://docs.python.org/3/library/configparser.html?highlight=configuration#supported-ini-file-structure).

May be we should use "," instead.